collections/_posts/2024-02-23-Rubiks-cube.md

```diff
@@ -0,0 +1,87 @@
+---
+layout: review
+title: "Self-supervised Feature Learning for 3D Medical Images by Playing a Rubik’s Cube"
+tags: self-supervised learning, medical imaging, 3D
+author: "Juliette Moreau"
+cite:
+    authors: "Xinrui Zhuang, Yuexiang Li, Yifan Hu, Kai Ma, Yujiu Yang, Yefeng Zheng"
+    title:   "Self-supervised Feature Learning for 3D Medical Images by Playing a Rubik’s Cube"
+    venue:   "MICCAI, 2019"
+pdf: "https://arxiv.org/pdf/1910.02241.pdf"
+---
+
+
+# Highlights
+
+* Presentation of a self-supervised pretext task for 3D medical images
+* Learning of translational and rotation invariant features from raw 3D data
+* Proven efficiency in classification or segmentation tasks
+
+# Introduction
+
+There is a lot of 3D data available in hospitals, but annotating it to train deep learning algorithms is very costly. One of the solutions proposed to take advantage of all this data is self-supervised learning. The goal is to pre-train a network with a large unannotated dataset, learning a pretext task, and then fine-tune it for the target task with the reduced annotated dataset.
+
+![](/collections/images/3D_Rubik/SSL.jpg)
+
+Classical pretext tasks are grayscale image colorization, inpainting or jigsaw puzzle among others. This article proposes for the first time a task specific to 3D data, inspired by the Rubik's cube.
+
+# Method
+
+## Data preprocessing
+
+The proposed task enforces the network to learn the translation and rotation invariant features from the raw data, as it involves both cube rearrangement and cube rotation. 
+
+![](/collections/images/3D_Rubik/method_diagram.jpg)
+
+The method is inspired by jigsaw puzzle pretext task but adapted for 3D medical images with an increased difficulty by adding some rotation.
+The first step is to separate the volume into cubes, which are then rearranged and randomly rotated. Since the goal is to learn high-level semantic features and not texture information close to cubes boundaries, a gap of "about 10 voxels" is left between two adjacent cubes. Each cube is also normalized to [-1,1].
+
+## Network architecture
+
+$$M$$ siamese networks ($$M$$ the number of cubes) with shared weight branches, that they call Siamese-Octad, are used to solve the problem. The backbone can be any 3D CNN. The feature maps of the last layer of all branches are concatenated and given to a last fully connected layer for the too tasks and supervised by permutation loss and rotation loss.
```

Comment:
Also I think a word (or a few words) are missing around _too_.

Reply:
It is a typo, I meant two and not too... I change that immediately